lib/yaml_normalizer/helpers/param_parser.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+module YamlNormalizer
+  module Helpers
+    # Methods handling passing of additional params from CLI
+    module ParamParser
+      # rubocop:disable Metrics/MethodLength
```

Comment:
I sense a new and exciting opportunity here ;-)
What would the code look like when making rubocop happy?